src/components/articles/articles-list/ArticlesList.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from "react";
+import classes from "./ArticlesList.module.css";
+import ArticleItem from "../article-item/ArticleItem";
+import { useState } from "react";
+import * as api from "../../../apis/articleApi";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const getArticles = () => {
+    api.fetchArticles().then((articlesData) => {
+      console.log(articlesData);
+      setArticles(articlesData);
+    });
+  };
+
```

Comment:
Only things missing

- [x] is loading state/message
- [x] error state/message